hazelcast/src/main/java/com/hazelcast/cache/impl/operation/CacheNearCacheStateHolder.java

```diff
@@ -58,7 +58,7 @@ void prepare(CachePartitionSegment segment, Collection<ServiceNamespace> namespa
         MetaDataGenerator metaData = getPartitionMetaDataGenerator(cacheService);
 
         int partitionId = segment.getPartitionId();
-        partitionUuid = metaData.getUuidOrNull(partitionId);
+        partitionUuid = metaData.getOrCreateUuid(partitionId);
```

Comment:
It looks like `getUuidOrNull` is no longer used except in test code, could you check if it is still needed?

Reply:
that's right, already checked it, it is only for testing now.